packages/mdc-chips/README.md

```diff
@@ -229,6 +234,7 @@ Method Signature | Description
 `deregisterInteractionHandler(evtType: string, handler: EventListener) => void` | Deregisters an event handler on the root element for a given event
 `createChipElement(text: string, leadingIcon: Element, trailingIcon: Element) => Element` | Returns a chip element with the given text, leading icon, and trailing icon
 `appendChild(el: Element) => void` | Appends the given element as a child of the root element
+`removeChip(chip: MDCChip) => void` | Removes the chip object from the chip set
```

Comment:
Do you think it would make sense to combine `createChipElement` and `appendChild` into a single `appendChip` API, to mirror `removeChip`? (I think those other two are only ever used together back to back anyway?)

Reply:
Yeah that's a really good idea